packages/komencikit/src/kit.ts

```diff
@@ -193,11 +210,11 @@ export class KomenciKit {
   })
   public async requestAttestations(
     identifier: string,
-    walletAddress: string,
+    metaTxWalletAddress: string,
```

Comment:
Nit: this is the only transaction wrapper that has `identifier` as the first argument and the address as the second. Can we swap these to avoid mistakes when calling?

Reply:
Yeah I actually thought of this as well I'll make all the function take the metaTxWalletAddress as the first argument.